src/helpers/single-value-providers.ts

```
@@ -0,0 +1,65 @@
+}
+
+export class CustomFieldIntegerValueProvider extends SingleValueProvider<number> {
+    constructor(public customField: string) {
+        super('custom field integer');
+    }
+
+    async getValue(): Promise<number | undefined> {
+        if (!this.customField) return undefined;
+
+        let participantInfo = await MyDataHelps.getParticipantInfo();
+        if (participantInfo.customFields.hasOwnProperty(this.customField)) {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)